### PR TITLE
fix rename merge

### PR DESCRIPTION
**Pull request by nicolaskruchten**

### CODE REVIEW

File: _posts/3d/2017-06-26-3d-line-plotly2.md [hunk 1]
```diff
@@ -21,16 +21,14 @@ live-graph: <iframe width="900" height="800" frameborder="0" scrolling="no" src=
 steps:
  - title: Try an Example
    sub-steps:
-    - copy: "To try an example, select 'Chart Type' and a panel will appear with chart type options. Once you locate the 3D line icon under the '3d'column, you can check out an example before adding your own data by clicking the little graph icon that will show what a sample chart looks like after adding data and playing with the style. You'll also see what labels and style attributes were selected for this specific chart, as well as the end result."
-      img: "![Try an Example](../static/images/create-animations/try-an-example.png)"
-    - copy: "You can also use the data featured in this tutorial by clicking on 'Open This Data in Chart Studio' on the left-hand side. It'll open in your workspace."
-      img: "![Open data](../static/images/enter-data-in-the-grid/open-this-data.png)"
+    - copy: "Before getting started with your own dataset, you can check out an example. First, select the 'Type' menu. Hovering the mouse over the chart type icon will display three options: 1) Charts like this by Plotly users, 2) View tutorials on this chart type, and, 3) See a basic example."
```
Comment (priyatharsan) [inline _posts/3d/2017-06-26-3d-line-plotly2.md]: Is it `Charts like this by Plotly users` or `Charts like this by Chart Studio users`

File: _posts/3d/2016-07-14-how-to-make-a-3D-scatter-plot.md [hunk 2]
```diff
@@ -22,18 +22,17 @@ live-graph-image:
...
-    - copy: "You can also use the data featured in this tutorial by clicking on 'Open This Data in Chart Studio' on the left-hand side. It'll open in your workspace."
-      img: "![Open data](../static/images/enter-data-in-the-grid/open-this-data.png)"
+    - copy: "Before getting started with your own dataset, you can check out an example. First, select the 'Type' menu. Hovering the mouse over the chart type icon will display three options: 1) Charts like this by Plotly users, 2) View tutorials on this chart type, and, 3) See a basic example."
+    - copy: "Clicking the 'See a basic example' option will show what a sample chart looks like after adding data and editing with the style. You'll also see what labels and style attributes were selected for this specific chart, as well as the end result."
+      img: "![Example after](../static/images/line-graph-and-scatter-plot-with-excel/scatter-try-example.gif)"
+    - copy: "You can also use the data featured in this tutorial by clicking on 'Open This Data in Plotly' on the left-hand side. It'll open in Chart Studio."
 
- - title: Add Your Data to Chart Studio
+ - title: Add Your Data to Plotly
```
Comment (priyatharsan) [inline _posts/3d/2016-07-14-how-to-make-a-3D-scatter-plot.md]: `Add Your Data to Chart Studio` ?

File: _posts/3d/2017-06-26-3d-line-plotly2.md (hunk 1 above)
Comment (nicolaskruchten) [reply _posts/3d/2017-06-26-3d-line-plotly2.md]: doesn't matter much.

Comment (priyatharsan) [reply _posts/3d/2017-06-26-3d-line-plotly2.md]: cool 👍

**Comment by priyatharsan** [comment]
💃

**Pull request merged**